viadot/sources/vid_club.py

```diff
@@ -80,70 +81,134 @@ def build_query(
             )
         return url
 
+    def intervals(
+        self, from_date: str, to_date: str, days_interval: int
+    ) -> Tuple[List[str], List[str]]:
+        """
+        Breaks dates range into smaller by provided days interval.
+
+        Args:
+            from_date (str): Start date for the query.
+            to_date (str): End date for the query. By default, datetime.today() will be used.
+            days_interval (int): Days specified in date range per api call (test showed that 30-40 is optimal for performance).
+
+        Returns:
+            List[str], List[str]: Starts abd Ends lists that contains information about date ranges for specific period and time interval.
+        """
+
+        if to_date == None:
```

Comment:
In this case maybe it could be good to add 'None' as default value to argument and include this information to docstring

Reply:
This function is used in total_load from which arguments are passed to intervals. If this will be used separately to create time intervals, maybe it's better not to have any default values.